roles/reset/tasks/main.yml

```diff
@@ -206,6 +206,7 @@
 
 - name: Clear IPVS virtual server table
   command: "ipvsadm -C"
+  ignore_errors: yes
```

Comment:
This is unrelated to this PR, best put it in another PR

Reply:
By mistake, it got added over here: 
Created another PR: https://github.com/kubernetes-sigs/kubespray/pull/7587

Reply:
You haven't removed the second commit